content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Migrate Docker Engine nodes from dockershim to cri-dockerd"
+weight: 9
+content_type: task 
+---
+
+{{% thirdparty-content %}}
+
+This page shows you how to migrate your Docker Engine nodes to use `cri-dockerd`
+instead of dockershim. Follow these steps if your clusters run Kubernetes 1.23
+or earlier and you want to continue using Docker as your container runtime after
+you upgrade to Kubernetes 1.24 and later.
+
+## Overview
+
+In Kubernetes 1.23 and earlier, Docker Engine used a component called the
+dockershim to interact with Kubernetes system components such as the kubelet. In
+Kubernetes 1.24, the dockershim is being removed. A third-party replacement,
```

Comment:
```suggestion
dockershim to interact with Kubernetes system components such as the kubelet.
The dockershim component is deprecated and will be removed in the Kubernetes v1.24
release. A third-party replacement,
```

Reply:
Maybe we should have two changes? One for now, and new wording for the v1.24 docs?

If the challenge is trying too hard to write timeless docs, the easier option might be to make multiple changes.